chore: DIA-2175: [TEMP] New button

nick-skriabin opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

nick-skriabin committed:
Button WIP - Primary variant

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/button

nick-skriabin committed:
Finalize styles and variants

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/button

nick-skriabin committed:
Add `buttonVariant` helper function

- `buttonVariant` to apply button styles to different elements
- Example of `buttonVariant` usage with `a` example

nick-skriabin committed:
Proper link

nick-skriabin committed:
Fix link preview

nick-skriabin committed:
Add Button to export list, organize exports

nick-skriabin committed:
Start using new Button

nick-skriabin committed:
Remove text shadow from outlined button

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Homepage fix button usage

nick-skriabin committed:
Replace "Create" button

nick-skriabin committed:
Add sizes and fix some UI bugs

nick-skriabin committed:
Fix button label paddings

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update buttons

nick-skriabin committed:
Fix sizing options

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update DM buttons

nick-skriabin committed:
Fix hover

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update buttons in project settings

nick-skriabin committed:
Update css formatting

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Simplify button layout for complex scenarios

nick-skriabin committed:
Updating button layout

nick-skriabin committed:
Icons alignment

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Updating buttons

nick-skriabin committed:
Fix button behavior when using outline/string look

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update hover behavior for outline/string looks

nick-skriabin committed:
Shadow behavior when focused/active

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
More buttons

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2156/button

nick-skriabin committed:
Use raw values instead of hardcoded ones

nick-skriabin committed:
Update color variable

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Remove LSO Button

nick-skriabin committed:
Replace buttons in Data Manager

nick-skriabin committed:
Update fonts

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Updating Editor buttons

nick-skriabin committed:
Icon sizing

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update task navigation buttons

nick-skriabin committed:
Smaller text size

nick-skriabin committed:
Allow overriding `disabled` state when waiting

nick-skriabin committed:
Fix neutral button focus outline

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Updating native buttons

nick-skriabin committed:
Fix Button usage in DM (caused error)

nick-skriabin committed:
Add ButtonGroup

nick-skriabin committed:
Merge branch 'fb-dia-2156/button' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Editor buttons

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Hotkeys

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix config labels

nick-skriabin committed:
Addressing CS feedback

nick-skriabin committed:
Update color

nick-skriabin committed:
Addressing feedback

nick-skriabin committed:
Remove traces

nick-skriabin committed:
Cleanup

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Biome fixes

nick-skriabin committed:
Update media controls

nick-skriabin committed:
Fix tests

nick-skriabin committed:
Update antd buttons (WIP)

nick-skriabin committed:
Replace antd buttons

nick-skriabin committed:
Update the rest of the buttons

nick-skriabin committed:
Biome

nick-skriabin committed:
Fix relations test

nick-skriabin committed:
Fix e2e and integration tests

nick-skriabin committed:
Fix selectors

nick-skriabin committed:
Biome

nick-skriabin committed:
Fix aria-labels

nick-skriabin committed:
Fix selectors

nick-skriabin committed:
Fix selector

nick-skriabin committed:
One more selectors fix

nick-skriabin committed:
Add proper aria-label to region delete button

nick-skriabin committed:
Restore "delete selected region" behavior

nick-skriabin committed:
Formatting

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Clarification on data-ignore-uikit

nick-skriabin committed:
Update buttons look

nick-skriabin committed:
Include libs in tailwind compilation process

nick-skriabin committed:
Adjust styles

nick-skriabin committed:
Fix button sizes

nick-skriabin committed:
Fix grid image preview buttons

nick-skriabin committed:
Fix save button in the API settings dialog

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix incorrect css

MihajloHoma committed:
Merge branch 'develop' into 'fb-dia-2142/new-button-lso'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14751552597

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14751552597

nick-skriabin committed:
Fix filter line and columns list

borisheartex committed:
Merge branch 'develop' into 'fb-dia-2142/new-button-lso'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14832954932

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14832954932

nick-skriabin committed:
Fix button story, remove global shadcn export

nick-skriabin committed:
Re-export Shadcn Badge

nick-skriabin committed:
Add missing exports

nick-skriabin committed:
Move button styles to tailwind class names

nick-skriabin committed:
Modal buttons

nick-skriabin committed:
Update controls look

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14906019023

borisheartex committed:
Merge branch 'develop' into 'fb-dia-2142/new-button-lso'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14906019023

nick-skriabin committed:
Adjust icon size to button's size

nick-skriabin committed:
Limit min-width for icon containers

nick-skriabin committed:
Fix border color for active state

nick-skriabin committed:
UI polishing

nick-skriabin committed:
Fix filter button

nick-skriabin committed:
Update button borders

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14965779129

nick-skriabin committed:
Adding aria-label attributes

nick-skriabin committed:
More aria-label

nick-skriabin committed:
Adding more aria-label attributes

nick-skriabin committed:
Add aria-label attributes to various buttons for improved accessibility

nick-skriabin committed:
Add aria-label attributes to confirm and info buttons for enhanced accessibility

nick-skriabin committed:
Add aria-label attributes to buttons in Debug, Annotations, and RegionItem components for improved accessibility

nick-skriabin committed:
Update aria-label attributes for buttons in Debug, Annotations, Comments, Taxonomy, and TopBar components to enhance accessibility and clarity

nick-skriabin committed:
Some more aria-label

nick-skriabin committed:
Custom reject button styles

nick-skriabin committed:
Cancel skip styles

nick-skriabin committed:
Update go to import button

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Add updated button component

nick-skriabin commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15002461003)

nick-skriabin committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15002461003

nick-skriabin committed:
Merge branch 'develop' into 'fb-dia-2175/new-button-lse'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15002461003

nick-skriabin committed:
Use correct button component

nick-skriabin committed:
Update Account Page buttons

nick-skriabin committed:
Add `cn` export

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15015026006

borisheartex committed:
Merge branch 'develop' into 'fb-dia-2142/new-button-lso'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15015026006

nick-skriabin committed:
Add waiting state to the button (useful for tests)

nick-skriabin committed:
Add waiting state to the button (useful for tests)

nick-skriabin committed:
Add variant state

nick-skriabin committed:
Add variant state

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15038127514

nick-skriabin committed:
Merge branch 'fb-dia-2142/new-button-lso' into fb-dia-2175/new-button-lse

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15064530699

nick-skriabin committed:
Restore aria-label for region actions

nick-skriabin committed:
Merge branch 'fb-dia-2142/new-button-lso' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Fix grid preview buttons

nick-skriabin committed:
Grid view buttons

nick-skriabin committed:
Larger icons

nick-skriabin committed:
Remove outline from region actions

nick-skriabin committed:
Gradient button

nick-skriabin committed:
Gradient button focused state

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15107153263

nick-skriabin committed:
Fix comment action button

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Biome fixes

nick-skriabin committed:
Biome checks

nick-skriabin committed:
Fix unit tests

nick-skriabin committed:
Restore proper aria-labels

nick-skriabin committed:
Update web/apps/labelstudio/src/pages/Organization/PeoplePage/InviteLink.tsx

Co-authored-by: Ignacio Velazquez <ignacio@humansignal.com>

nick-skriabin committed:
Update web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.jsx

Co-authored-by: Ignacio Velazquez <ignacio@humansignal.com>

nick-skriabin committed:
Replace Elem buttons with Button

nick-skriabin committed:
Merge branch 'fb-dia-2142/new-button-lso' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Fix integration tests

nick-skriabin committed:
Update integration tests to match selectors

nick-skriabin committed:
Cleanup

nick-skriabin committed:
Update integration tests

nick-skriabin committed:
Update taxonomi per-tag MIG

nick-skriabin committed:
Remove unnecessary comment

nick-skriabin committed:
Remove old code (regression from develop)

nick-skriabin committed:
Replace native buttons, remove tooltip component

nick-skriabin committed:
Replace native button

nick-skriabin committed:
Remove `primary` prop

nick-skriabin committed:
Use TW class

nick-skriabin committed:
Remove conflict resolution artifacts

nick-skriabin committed:
Merge branch 'fb-dia-2142/new-button-lso' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

yyassi-heartex committed:
Update label_studio/core/static/css/login.css

nick-skriabin committed:
Restore size prop for instruments

nick-skriabin committed:
Update web/libs/editor/src/core/Hotkey.ts

Co-authored-by: Andrew <hlomzik@gmail.com>

nick-skriabin committed:
Remove unnecessary tooltip

nick-skriabin committed:
Update web/libs/editor/src/components/SidePanels/OutlinerPanel/ViewControls.tsx

Co-authored-by: Andrew <hlomzik@gmail.com>

nick-skriabin committed:
Update web/libs/editor/src/components/BottomBar/buttons.tsx

Co-authored-by: Andrew <hlomzik@gmail.com>

nick-skriabin committed:
Update web/libs/editor/src/components/BottomBar/buttons.tsx

Co-authored-by: Andrew <hlomzik@gmail.com>

nick-skriabin committed:
Update web/apps/labelstudio/src/pages/Projects/ProjectsList.jsx

Co-authored-by: Andrew <hlomzik@gmail.com>

nick-skriabin committed:
Various tiny fixes

nick-skriabin committed:
Restore transform

nick-skriabin committed:
Remove excess background property

nick-skriabin committed:
Use variable for font size

nick-skriabin committed:
Use token for text color

nick-skriabin committed:
Fine tuning

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Update sample remove button

nick-skriabin committed:
Dark mode fixed

nick-skriabin committed:
Fix border radius on login page inputs

nick-skriabin committed:
Fix modal close button

nick-skriabin committed:
Fix danger zone buttons

nick-skriabin committed:
Fix machine learning ellipsis button

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix config delete label button

nick-skriabin committed:
Fix grid view toggle and buttons

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15591774620

nick-skriabin committed:
Fix gradient button animation

nick-skriabin committed:
Merge branch 'fb-dia-2175/new-button-lse' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Tailwind config develop

nick-skriabin committed:
Remove outline from the modal close button

nick-skriabin committed:
Add aria label to close modal button

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'fb-dia-2175/new-button-lse' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Add aria-label to the actions button

nick-skriabin committed:
Add aria-label to task action buttons

nick-skriabin committed:
Fix aria labels

nick-skriabin committed:
Remove console.log

nick-skriabin committed:
Fix modal window not closing

nick-skriabin committed:
Fix modal not closing

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

borisheartex commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16025397476)

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16025397476

borisheartex committed:
Merge branch 'develop' into 'fb-dia-2175/new-button-lse'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16025397476

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix icon import

nick-skriabin committed:
Fix filter icons

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16068319129

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix tests

nick-skriabin committed:
Custom hotkeys button

nick-skriabin committed:
Fix references

nick-skriabin committed:
Merge branch 'develop' into 'fb-dia-2142/new-button-lso'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16120193272

nick-skriabin committed:
Tasks

nick-skriabin committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16120785427

nick-skriabin committed:
Fix tests

nick-skriabin committed:
Fix test

nick-skriabin committed:
Enable tasks

nick-skriabin committed:
Restore screenshot comparison

nick-skriabin committed:
Fix controls

nick-skriabin committed:
Fix tests

nick-skriabin committed:
Fix last test

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin committed:
Fix audio paragraphs test

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2142/new-button-lso

nick-skriabin commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16171243935)
> Successfully merged: Merge made by the 'ort' strategy.

nick-skriabin committed:
Merge branch 'develop' into 'fb-dia-2175/new-button-lse'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16171243935

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse

nick-skriabin committed:
Merge branch 'develop' into fb-dia-2175/new-button-lse
